README.md

```diff
@@ -197,13 +191,6 @@ If you like this project please follow me on github & twitter and also connect o
 ## :pray: Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- [OPTIONAL] Excellent job documenting a  good README 🥇 I think it would be better if you can fix the headers of these sections. It seems that although you are using `##` for creating a heading, but I think it is not working well. Please see the images below 🔽 Please consider fixing these headings to make the README even more professional 💪⚒

![image](https://user-images.githubusercontent.com/106119105/219355169-32eb72e3-b14a-42df-b44f-31dff24d1f9b.png)